ci_framework/roles/switch_config/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+# Amount of retries to supported tasks
```

Comment:
All variables specific to this role should have a prefix of  `cifmw_switch_config`